app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+class Post < ApplicationRecord
+  default_scope -> { order(created_at: :desc) }
+
+  validates :content, presence: true, length: { maximum: 1000 }
+  validates :user, presence: true
```

Comment:
We don't need to validate anything that is already validated by the `belongs_to` association.